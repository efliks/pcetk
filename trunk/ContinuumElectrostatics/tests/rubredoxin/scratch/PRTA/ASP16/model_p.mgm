ON_GEOM_CENT 121 2.000000
(23.414162 33.054197 9.397780) 101 1.000000
(23.414162 33.054197 9.397780) 101 0.500000
(23.414162 33.054197 9.397780) 101 0.250000
