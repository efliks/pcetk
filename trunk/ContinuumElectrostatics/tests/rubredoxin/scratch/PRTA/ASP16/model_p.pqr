ATOM      0  C   GLU    15      19.348  32.868   8.562   0.51000   1.700      0      0      0      0
ATOM      0  O   GLU    15      18.767  32.167   9.386  -0.51000   1.500      0      0      0      0
ATOM      0  N   ASP    16      20.576  33.345   8.746  -0.47000   1.550      0      0      0      0
ATOM      0  H   ASP    16      21.005  33.961   8.084   0.31000   0.300      0      0      0      0
ATOM      0  CA  ASP    16      21.332  33.017   9.961   0.07000   1.700      0      0      0      0
ATOM      0  HA  ASP    16      21.290  31.942  10.102   0.09000   0.300      0      0      0      0
ATOM      0  CB  ASP    16      22.799  33.437   9.876   0.00000   1.700      0      0      0      0
ATOM      0  HB1 ASP    16      23.279  33.339  10.875   0.00000   1.000      0      0      0      0
ATOM      0  HB2 ASP    16      22.877  34.494   9.553   0.00000   1.000      0      0      0      0
ATOM      0  CG  ASP    16      23.627  32.573   8.936   0.00000   1.700      0      0      0      0
ATOM      0  OD1 ASP    16      23.376  31.357   8.870   0.00000   1.500      0      0      0      0
ATOM      0  OD2 ASP    16      24.527  33.125   8.277   0.00000   1.500      0      0      0      0
ATOM      0  C   ASP    16      20.689  33.621  11.230   0.51000   1.700      0      0      0      0
ATOM      0  O   ASP    16      20.752  33.003  12.304  -0.51000   1.500      0      0      0      0
ATOM      0  N   ALA    17      20.051  34.789  11.117  -0.47000   1.550      0      0      0      0
ATOM      0  H   ALA    17      19.993  35.217  10.216   0.31000   0.300      0      0      0      0
ATOM      0  CA  ALA    17      19.432  35.497  12.233   0.07000   1.700      0      0      0      0
ATOM      0  HA  ALA    17      19.944  35.246  13.152   0.09000   0.300      0      0      0      0
