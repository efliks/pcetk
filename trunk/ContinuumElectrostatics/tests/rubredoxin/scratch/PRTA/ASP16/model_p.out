Starting /home/mikolaj/local/bin//my_2diel_solver for molecule named scratch/PRTA/ASP16//site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:16:CB",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:16:CG",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:16:HB1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:16:HB2",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:16:OD1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:16:OD2",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (21.3147, 33.4999, 10.0822)) 121 2
Finer level:    (23.4142, 33.0542, 9.39778) 101 1
Finer level:    (23.4142, 33.0542, 9.39778) 101 0.5
Finest level:   (23.4142, 33.0542, 9.39778) 101 0.25


Self energy of scratch/PRTA/ASP16//site_p in two dielectric environment = 233.325
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/ASP16//site_p with background charge set in two dielectric environment = -0.0851891
(probably in kcal/mole)
Total runtime: 14.04 s user 0.1 s system 
